website/static/js/fangorn.js

```diff
@@ -532,8 +546,22 @@ function doItemOp(operation, to, from, rename, conflict) {
  * @private
  */
 function _fangornResolveUploadUrl(item, file) {
+    // WB v1 update synatx is file path and kind=file
+    // WB v1 upload synatx is parent path and kind=file, name=filename
+    // If upload target file name already exists don't pass file.name
+    var updateUrl;
```

Comment:
Minor: since `updateUrl` isn't used when `configOption` exists, don't declare it until after that branch.  IOW move this declaration to right before the `$.each()`.